modules/custom/openy_upgrade_tool/src/OpenyUpgradeLogManager.php

```diff
@@ -0,0 +1,588 @@
+<?php
+
+namespace Drupal\openy_upgrade_tool;
+
+use Drupal\config\Form\ConfigSync;
+use Drupal\config\StorageReplaceDataWrapper;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigImporter;
+use Drupal\Core\Config\ConfigImporterException;
+use Drupal\Core\Config\ConfigManagerInterface;
+use Drupal\Core\Config\StorageComparer;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Config\TypedConfigManagerInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Extension\ModuleInstallerInterface;
+use Drupal\Core\Extension\ThemeHandlerInterface;
+use Drupal\Core\Lock\LockBackendInterface;
+use Drupal\Core\Render\RendererInterface;
+use Drupal\Core\Url;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\features\FeaturesManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Logger\LoggerChannelInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+/**
+ * Wrapper for main logic that related to OpenyUpgradeLog.
+ */
+class OpenyUpgradeLogManager implements OpenyUpgradeLogManagerInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * Upgrade path dashboard route.
+   */
+  const DASHBOARD = 'entity.openy_upgrade_log.collection';
+
+  /**
+   * Upgrade path dashboard route.
+   */
+  const MODAL_WIDTH = 1000;
```

Comment:
what about mobile view?
Can we somehow make this via CSS?

Reply:
It looks fine on mobile view, modal is responsive.

![screenshot from 2019-01-23 12-09-30](https://user-images.githubusercontent.com/13733670/51599288-ea70bd80-1f07-11e9-9ec0-7f4293fa5560.png)